content/reference/sql/benchmarks/correctness.md

```diff
@@ -68,3 +69,22 @@ did not run could not complete due to a timeout earlier in the run.
 |------------------------|-----------|
 <!-- END___DOLT___CORRECTNESS_RESULTS_TABLE -->
 <br/>
+
+## Function Coverage
+We also measure the coverage of the functions in the SQL engine. This
+is a measure of how many of the supported MySQL functions are also
+supported by Dolt. 
+
+Here are Dolt's function coverage results for version `1.34.3`.
+| Supported | Total | Percent Coverage |
+|-----------|-------|------------------|
+|       263 |   438 |        60.045662 |
+
+## Skipped Engine Tests
+Here are the total number of tests skipped by the engine for 
+version `1.34.3`. These are tests that we know are failing for one
+reason or another, but haven't been able to fix yet.
+
```

Comment:
I think a paragraph here about why these are harder to fix then sqllogictests:

1. Constructed by us as we built Dolt. Human identified edge cases.
2. Not watered down by passing tests that test the same thing
3. Augment areas not caught by sqllogictest